1-JavaScript/Week4/homework/ex3-lemonAllergy.test.js

```diff
@@ -26,26 +26,28 @@ const fruitBasket = [
 ];
 
 // ! Function under test
-function sanitizeFruitBasket(/* TODO parameter(s) go here */) {
-  // TODO complete this function
+function sanitizeFruitBasket(currentBasket, unwantedFruit) {
+  const healthyBasket = currentBasket.filter(
```

Comment:
```suggestion
  const sanitizedBasket = currentBasket.filter(
```
Not sure what healthy means in this context

Reply:
I made the edit